gradle/verification-metadata.xml

```diff
@@ -1342,9 +1342,9 @@
             <pgp value="2e3a1affe42b5f53af19f780bcf4173966770193"/>
          </artifact>
       </component>
-      <component group="org.jetbrains" name="annotations" version="24.0.0">
-         <artifact name="annotations-24.0.0.jar">
-            <pgp value="33fd4bfd33554634053d73c0c2148900bcd3c2af"/>
+      <component group="org.jetbrains" name="annotations" version="24.0.1">
+         <artifact name="annotations-24.0.1.jar">
+            <sha256 value="61666dbce7e42e6c85b43c04fcfb8293a21dcb55b3c80e869270ce42c01a6b35" origin="Generated by Gradle"/>
```

Comment:
@JT122406 Please, leave pgp value here, that way it would pass dependency verification.

Reply:
whoops my bad I will fix later when at computer